docs/conf.py

```diff
@@ -106,14 +100,6 @@
 html_static_path = ["_static"]
 
 
-# These paths are either relative to html_static_path
```

Comment:
this is the gitter chat box that is no longer available... now that gitter was migrated to matrix